incubator/prometheus/values.yaml

```diff
@@ -1,10 +1,200 @@
-image: prom/prometheus
-imageTag: v1.1.0
-imageAlertManager: prom/alertmanager
-imageTagAlertManager: v0.4.2
-
-# Persist data to a persitent volume
-persistence:
-  enabled: true
-  storageClass: generic
-  size: 8Gi
+alertmanager:
```

Comment:
This is really nicely done. Love the formatting, documentation and configurable options. 👏